src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -404,19 +404,24 @@
     <Field Name="Right" Type="BoundExpression"/>
 
     <Field Name="DeconstructSteps" Type="ImmutableArray&lt;BoundDeconstructionDeconstructStep&gt;" Null="NotApplicable" SkipInVisitor="true" />
```

Comment:
>Field [](start = 5, length = 5)

It is a little late for me to make this comment, but I feel like too much of the lowering strategy has crept into the shape of the bound nodes. At some future time I'd like us to revisit the bound node shape for these. Part of that probably involves introducing into the tuple conversion object a way of getting the underlying (elementwise) conversions. That compound tuple conversion certainly does deserve to be stored on this assignment operator.